extensions/ql-vscode/src/language-context-store.ts

```diff
@@ -46,4 +46,11 @@ export class LanguageContextStore extends DisposableObject {
   public shouldInclude(language: QueryLanguage | undefined): boolean {
     return this.languageFilter === "All" || this.languageFilter === language;
   }
+
+  public selectedLanguage(language: QueryLanguage | undefined): boolean {
+    return (
+      (this.languageFilter === "All" && language === undefined) ||
```

Comment:
Does this make sense in all contexts where this method may be used? I can imagine in other contexts that when all languages are selected, it should return `true` when the `language` is any value.

Reply:
I think that is what the `shouldInclude` method does. I'll add comments to both of them to make the usage more clear.